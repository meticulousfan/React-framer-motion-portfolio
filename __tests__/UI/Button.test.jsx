/**
 * @jest-environment jsdom
 */

import Button from "../../src/components/UI/Button.component";
import { render } from "@testing-library/react";

const handleOnClick = jest.fn();

describe("Button", () => {
  it("Button laster inn og kan vises", () => {
    const { container } = render(<Button onClick={handleOnClick}>Button</Button>);
    expect(container).toMatchSnapshot();
  });
});
