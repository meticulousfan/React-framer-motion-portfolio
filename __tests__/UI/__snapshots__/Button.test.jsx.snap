// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Button laster inn og kan vises 1`] = `
<div>
  <button
    class="p-4 m-4 text-white transition duration-500 ease-in-out bg-gray-800 rounded hover:shadow-outline hover:bg-gray-600"
    type="submit"
  >
    Button
  </button>
</div>
`;
